### ISSUE TITLE
Hacktoberfest 2018?

### ISSUE THREAD

**Issue by Psypher9** [restsharp/RestSharp#1177]
I'm curious to know, will RestSharp be participating in [Hacktoberfest](https://hacktoberfest.digitalocean.com/) this year?

**Comment by Psypher9** [comment]
Yeah, when I had asked that question, they had a spot for Maintainers, but reading through the participation rules it looks to me like Hacktoberfest will count for all repositories. There are just some groups, like MSFT, that are doing some other participation events. I was a little confused before too.

**Comment by alexeyzimarev** [comment]
I remember that LNUG did something last year and I remember that .NET Foundation claims to help with some organisation around this. However, I tried to bring RestSharp to be a part of the .NET Foundation twice and got no reply from them whatsoever. Quite disappointing.
